example/app/src/main/java/com/rootstrap/androidcomposebase/ui/pages/settings/SettingsScreen.kt

```diff
@@ -170,7 +179,10 @@ private fun SettingsScreenPreview() {
     val boolean = isSystemInDarkTheme()
     var darkTheme by remember { mutableStateOf(boolean) }
     AppTheme {
-        SettingsScreen(isOSDarkTheme = darkTheme,
-            onThemeUpdated = { darkTheme = !darkTheme })
+        SettingsScreen(
+            email = "rs@rs.com",
```

Comment:
is this supposed to be a placeholder? If true i'd suggest to move it into a constant otherwise maybe we can leave it empty for now

Reply:
fixed